flocker/node/agents/_logging.py

```diff
@@ -7,12 +7,8 @@
 
 from eliot import Field, ActionType, MessageType
 
-# Begin: Helper datastructures to log all
-# IBlockDeviceAPI calls from AWS storage driver using Eliot.
-# - Log all IBlockDeviceAPI calls as Eliot ``ActionType``.
-# - Log ``EC2ResponseError`` from AWS to Boto.
-# - Log failure of attached device manifestation
-# - Log OS out of available devices for attaching volume.
+# Begin: Common structures used by all (AWS, OpenStack)
+# storage drivers.
```

Comment:
Thanks for making me evaluate the logs. The log structures are not yet large in number to deserve a separate module for each storage driver. Trimmed down the documentation.